content/docs/instrumenting/exporter_guidelines.md

```diff
@@ -112,55 +114,69 @@ The first is some label it’d be useful to have on the metrics that are about,
 
 The other case are what are really target labels. These are things like region, cluster names, and so on, that come from your infrastructure setup rather than the application itself. It’s not for an application to say where it fits in your label taxonomy, that’s for the person running the Prometheus server to configure and different people monitoring the same application may give it different names.
 
-Accordingly these labels belong up in the scrape configs of Prometheus via whatever service discovery you’re using. It’s okay to apply [http://www.robustperception.io/how-to-have-labels-for-machine-roles/](http://www.robustperception.io/how-to-have-labels-for-machine-roles/) here too, as it’s likely useful information for at least some of the people scraping it.
+Accordingly these labels belong up in the scrape configs of Prometheus via whatever service discovery you’re using. It’s okay to apply the linked principe here too, as it’s likely useful information for at least some of the people scraping it.
```

Comment:
It's already linked in the same section just two sentences above. Following the wikipedia principle, only the first occurrence gets linked. I'll try to rewrite the sentence; I had to read the full thing a couple of times until I started to understand what it is about.